tests/uijson_test.py

```diff
@@ -93,23 +93,23 @@ def test_gravity_uijson(tmp_path):
     uijson = GravityInversionUIJson.read(params_uijson_path)
     uijson_path = tmp_path / "from_uijson.ui.json"
     uijson.write(uijson_path)
-    with open(params_uijson_path, encoding="utf-8") as f:
-        params_data = json.load(f)
-    with open(uijson_path, encoding="utf-8") as f:
-        uijson_data = json.load(f)
-
-    params_data_nobraces = {}
-    for param, data in params_data.items():
-        if isinstance(data, dict):
-            field_data_nobraces = {}
-            for field, value in data.items():
-                if isinstance(value, str):
-                    value = value.removeprefix("{").removesuffix("}")
-                if isinstance(value, list):
-                    value = [v.removeprefix("{").removesuffix("}") for v in value]
-                field_data_nobraces[field] = value
-        else:
-            field_data_nobraces = data
-        params_data_nobraces[param] = field_data_nobraces
-
-    assert uijson_data == params_data_nobraces
+    # with open(params_uijson_path, encoding="utf-8") as f:
+    #     params_data = json.load(f)
+    # with open(uijson_path, encoding="utf-8") as f:
+    #     uijson_data = json.load(f)
```

Comment:
was this not passing anymore?

Reply:
I will revert. I was testing the warnings, and looks like uijson.write throws a lot of warnings. Will need to investigate.